hal/nxp_p1021.c

```diff
@@ -906,6 +1010,20 @@ static int hal_cpld_init(void)
     /* Setup TLB MMU (Translation Lookaside Buffer) for CPLD/BCSR */
     set_tlb(1, 8, BCSR_BASE, BCSR_BASE, MAS3_SX | MAS3_SW | MAS3_SR,
         MAS2_I | MAS2_G, 0, BOOKE_PAGESZ_256K, 1);
+
+    /* setup eLBC for CPLD (CS1), 8-bit */
+    set32(ELBC_BR(1), (ELBC_BR_ADDR(BCSR_BASE) | ELBC_BR_MSEL(0) |
+        ELBC_BR_PS(1) | ELBC_BR_V));
+    set32(ELBC_OR(1), (ELBC_OR_AMASK(ELBC_AMASK_32KB) |
+        ELBC_ORG_CSCT | ELBC_ORG_XACS | ELBC_ORG_SCY | ELBC_ORG_TRLX |
+        ELBC_ORG_EHTR | ELBC_ORG_EAD));
+
+#if 0 /* ethernet (uec) setup not required */
```

Comment:
Remove dead code or add another define to enable ethernet initialization from the config when appropriate